### ISSUE TITLE
New UI: Quick Look support

### ISSUE THREAD

**Issue by dan-el** [strongbox-password-safe/Strongbox#590]
Currently pressing the spacebar in an entry's details section in the right column activates the "three dot" menu requiring an "extra click" if I wanted to let's say preview a PDF document.
![image](https://user-images.githubusercontent.com/19776691/152633487-d52519c5-dcc4-4ed4-96b2-75e9075a50d2.png)

Can we have the same spacebar behaviour in Strongbox that exists in macOS? which would be to preview the attachment directly.
I can see that the Quick Look feature (if possible) could also work on the "Notes" field which might be useful with huge notes.

**Comment by strongbox-mark** [comment]
Interesting, but if you hit Enter you should see the preview correct?

The thing is, we've designed things so that you can use the keyboard for easy navigation, and the space bar on all fields brings up the context menu, Enter performs a "Default" action (preview in the case of Attachments).

I'm not sure I'm crazy about shifting changing that.

**Comment by dan-el** [comment]
On attachments like PDF I can use "Enter" to open Quick Look from within Strongbox, but then it's either "Clicking the X" or hitting "Spacebar" to close it. While this works it's not intuitive. I'd much rather prefer the system-wide shortcuts to work within Strongbox and choose a different key than "Spacebar" to open the context menu.

Hitting "Enter" on the "Notes" filed in Strongbox copies the notes. Same on all other fields, except for attachments.
And in turn "Spacebar" does not open the context menu on the "Notes" field.

I see several actions which need to be tied to shortcuts/keys:
* Quick Look (I vote for "Spacebar")
   * This could preview various attachment types, maybe URLs in a web view, and excessive text notes
* Copy contents/attachment (CMD+C maybe? Though that is Stronbox-wide "copy password" which makes sense...)
   * Currently "Enter", except for attachments where "Enter" performs quick look
* Open context menu (enter?)
   * Currently "Spacebar" except on the "Notes" field.
* Execute (cmd/shift+enter maybe?)
   * Execute would mainly launch URLs in the default browser, or launch other apps in case another scheme is used

TL/DR: I think shortcuts/keys should be consistent across fields and if possible similar/same to system-wide macOS shortcuts.

**Comment by dan-el** [comment]
I love it, thanks a lot, Mark!
